server/src/main/java/org/threadly/litesockets/server/http/HTTPServer.java

```diff
@@ -95,6 +93,27 @@ public int getListenPort() {
     return port;
   }
   
+  /**
+   * Allows you to set/reset the ssl context on the server.  Good for dynamically reloading certs.
+   * 
+   * @param sslc The sslContext to use.
+   */
+  public void setSSLContext(final SSLContext sslc) {
+    this.sslc = sslc;
+    if(this.sslc != null) {
+      server.setSSLContext(this.sslc);
+      server.setSSLHostName(hostname);
+      server.setDoHandshake(true);
+    } else {
+      server.setDoHandshake(false);
+      server.setSSLContext(null);
+    }
+  }
+  
+  public void setExceptionHandler() {
+    
```

Comment:
Missing implementation?

Reply:
removed that was moved into the bodyListener.  Looking at it more, gonna move it into the generic handler since it could happen at any point in the parsing.  Also gonna close the client once it happens.